src/applications/representative-search/components/results/SearchResultsHeader.jsx

```diff
@@ -74,6 +75,7 @@ export const SearchResultsHeader = props => {
       {props.onClickApplyButtonTester ? (
         <button
```

Comment:
I wonder if there's a specific reason that you're not able to use the `va-button` web component? The `button` styles will eventually be deprecated from Formation so the sooner those can get updated, the easier will be down the road.

Reply:
This button is only used for testing - the actual UI button is a va-button, but I was having issues triggering it with enzyme (I had similar issues triggering checkboxes in that shadow-root). 

I just tried the following, which didn't seem to get picked up:

`wrapper.find('va-button[text="Apply"]').simulate('click');`